third_party/gfxreconstruct/framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -730,13 +730,21 @@ void VulkanReplayConsumerBase::ProcessCreateHardwareBufferCommand(
         stride = width;
     }
 
+    // GOOGLE: workaround to replay compositor capture
+    if (format > AHARDWAREBUFFER_FORMAT_S8_UINT || usage == 0x30122)
```

Comment:
interesting, do you know the actual format that they are using in this case? what is 0x30122?